api/api/docs/oauth2_docs.py

```diff
@@ -57,7 +56,6 @@
     request={"application/x-www-form-urlencoded": OAuth2TokenRequestSerializer},
     res={
         200: (OAuth2TokenSerializer, auth_token_200_example),
-        401: (NotAuthenticated, None),
```

Comment:
This should probably be added back for the "invalid_client" error.

Reply:
Thanks for finding these solutions! I'll push them next week, drafting this for now.